src/App.js

```diff
@@ -1,23 +1,85 @@
-import logo from './logo.svg';
+// App.js
```

Comment:
Maybe as a stretch goal, the user can filter the results by genre. Otherwise GREAT APP! Will definitely use to see the synopsis of a movie!!!! 🎦